native/android/app/build.gradle

```diff
@@ -79,7 +79,7 @@ import com.android.build.OutputFile
 
 project.ext.react = [
         extraPackagerArgs: (System.getenv('EXTRA_PACKAGER_ARGS') ?: "").split("\\s+"),
-        enableHermes: false,  // clean and rebuild if changing
+        enableHermes: true,  // clean and rebuild if changing
```

Comment:
There is a task for that. That means we probably need to check a few settings: https://issues.tuerantuer.org/browse/IGAPP-163

If you just did this for debugging then disable it again before merging.